modules/splitter/monitoring-the-solver-proc.adoc

```diff
@@ -134,3 +134,46 @@ of the measuring.
 * `optaplanner.solver.solve-length.seconds-max`: run time of the
 longest-running currently active solver.
 * `optaplanner.solver.solve-length.seconds-duration-sum`: the sum of each active solver's solve duration. For example, if there are two active solvers, one running for three minutes and the other for one minute, the total solve time is four minutes.
+
+== Additional Metrics
+
+For more detailed monitoring, you can configure {PLANNER_SHORT} to monitor additional metrics at a performance cost. The following example uses the `BEST_SCORE` and `SCORE_CALCULATION_COUNT` metric:
```

Comment:
You give an XML example, but you never tell the user into what file to place this XML into. I *guess* this is the solver source but please double check that. The user needs to know where this goes.